### ISSUE TITLE
Enabled wallpanel and swipe doesnt work for me.

### ISSUE THREAD

**Issue by cracyfloyd** [j-a-n/lovelace-wallpanel#285]
I Like to use swipe to switch through my different dashboards. But it doesnt work if i enable the wallpanel funktion. Anyone others have the same Problem? Or can help me !!

**Comment by j-a-n** [comment]
Hi,
do you mean https://github.com/zanna-37/hass-swipe-navigation ?

**Comment by cracyfloyd** [comment]
Yes thats the plugin i like to use in kombination with the Wall Panel… But for me it doesnt work both together.

**Comment by j-a-n** [comment]
Works for me.
What device / app / browser are you using?

**Comment by j-a-n** [comment]
Since there has been no recent activity on this topic, I am closing it.
If there is new information, the ticket can be reopened.
